solutions/usaco-491.mdx

```diff
@@ -2,14 +2,67 @@
 id: usaco-491
 source: USACO Silver 2014 December
 title: Piggy Back
-author: Qi Wang
+author: Qi Wang, Kakulavaram Sanjana Reddy
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_piggyback_silver.html)
 
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+
+<CPPSection>
+```cpp
+	
+#include <bits/stdc++.h>
+using namespace std;
+unordered_map<int, vector<int>> mp;
+vector<vector<int>> dist;
+void bfs(int s, int ix)
```

Comment:
what does this function do?

Reply:
Finds the distance between node s and node ix, I have included a comment defining this in code

Reply:
can you review the code please.